completions/evince

```diff
@@ -25,7 +25,7 @@ _evince()
         return
     fi
 
-    _filedir '@(@(?(e)ps|?(E)PS|[pf]df|[PF]DF|dvi|DVI)?(.gz|.GZ|.bz2|.BZ2|.xz|.XZ)|cb[rz]|djv?(u)|tif?(f))'
+    _filedir '@(@(?(e|?(o)x)ps|?(E)PS|[pf]df|[PF]DF|dvi|DVI)?(.gz|.GZ|.bz2|.BZ2|.xz|.XZ)|cb[7rtz]|djv?(u)|tif?(f))'
```

Comment:
I have tried to open a `.oxps.gz` file with `evince` (evince-42.2 on Fedora 36), but it doesn't seem to support `.oxps.gz` while it does support `.ps.gz` and `.pdf.gz`